material-github.user.css

```diff
@@ -1303,12 +1303,15 @@
  * | ==========================================
  */
 @-moz-document domain("render.githubusercontent.com"){
-	body[data-render-url="https://render.githubusercontent.com"]{
```

Comment:
Why did you remove this part? I think you should just add `, #notebook`.

Reply:
I dunno, I tried changing the background color for the body tag, but somehow nothing changed so I just replaced it with `#notebook`.

Reply:
please use `body[data-render-url="https://render.githubusercontent.com"], #notebook` instead, since it would break other pages if you remove the `body[...` part